docs/getting-started-single-cluster.md

```diff
@@ -51,7 +51,7 @@ If you want to make use of the Kuadrant `DNSPolicy` you should setup the followi
 ### Set the release you want to use 
 
 ```bash
-export KUADRANT_REF=main
+export KUADRANT_REF=v1.0.0
```

Comment:
I would prob just set this to ```release-v1.0.0``` that way we can fix minor issues with the scripts etc

Reply:
makes sense